packages/app/cypress/e2e/cypress-in-cypress-e2e.cy.ts

```diff
@@ -148,7 +148,7 @@ describe('Cypress In Cypress E2E', { viewportWidth: 1500, defaultCommandTimeout:
   })
 
   it('shows a compilation error with a malformed spec', { viewportHeight: 596, viewportWidth: 1000 }, () => {
-    const expectedAutHeight = 500 // based on explicitly setting viewport in this test to 596
+    const expectedAutHeight = 456 // based on explicitly setting viewport in this test to 596
```

Comment:
I changed the `<a>` for the AUT URL to be an `<input`> so we can type into it to navigate to a URL for Cypress Studio, and for some reason the wrapping break points for `flex-wrap` is _slightly_ different. I have no idea why and was quite stuck on this, but for all intensive purposes the UI is the same, except things wrap in the AUT header slightly earlier if you have your Cypress window really small. 

We can probably revisit this in the final pass if needed, but I don't think it makes any difference.